src/Joomlatools/Console/Command/Site/Listing.php

```diff
@@ -91,12 +99,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        $i = 1;
-        foreach ($sites as $site) {
+        if (!in_array($input->getOption('format'), ['txt', 'json'])) {
```

Comment:
We don't set a minimum PHP version requirement anywhere. The shorthand array syntax is only available in PHP 5.4. The box can also run 5.3. We should replace this with the `array('txt, 'json')` notation for now so we don't have to set the restrictions?